index.html

```diff
@@ -741,5 +742,36 @@
     <category name="Block Library" colour="260" id="blockLibCategory"></category>
   </xml>
 
+<!-- New Block Popup -->
+<div id="popup">
+  <div id="bkg"></div>
+  <div id="box">
+    <div id="exit">x</div>
+    <h1>Create a New Block</h1>
+    <form type=POST>
```

Comment:
We aren't sending this form anywhere, so POST is probably wrong.  I'm not even sure <form> is necessary for such component interactions handled purely by JavaScript.

Further, if did need it, we would still need to quote the attribute for style guide reasons.